### PR TITLE
2D semi-automated contouring

### PR DESCRIPTION
**Pull request by pedrooste**
Adding a Breadth First Search (BFS) functionality to the draw (fill) button for the 2D semi-automated contouring. This will draw the ROI for the single slice (2D) with a full 3D fill to come later.

### CODE REVIEW

File: src/View/mainpage/DrawROIWindow/Drawing.py [hunk 1]
```diff
@@ -153,24 +165,17 @@ def update_pixel_transparency(self):
         self.refresh_image()
         logging.debug("update_pixel_transparency finished")
 
-    def _find_neighbor_point(self, event):
```
Comment (pedrooste) [inline src/View/mainpage/DrawROIWindow/Drawing.py:156]: redundant method

**Review by nwilde — APPROVED** [review]
Looks good 👍

**Review by SeanOverton — APPROVED** [review]
Quality

**Comment by djwfdev** [comment]
Great work! no problems from me

**Pull request merged**